src/components/ChatLog.jsx

```diff
@@ -0,0 +1,39 @@
+import './ChatLog.css';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+
+const ChatLog = ({entries, onLikeToggle}) => {
+  return (
+    <div className='chat-log'>
+      <ul>
+        {entries.map((entry) => (
+          <ChatEntry
+            key={entry.id}
+            id={entry.id}
+            sender={entry.sender}
+            body={entry.body}
+            timeStamp={entry.timeStamp}
+            side={entry.sender === 'Vladimir' ? 'local' : 'remote'}
```

Comment:
For this line of code, I think that it could be moved inside `ChatEntry.jsx` is since it is for styling that component. I would be a single line with the ternary assigned to a variable before the `return` statement. 
```js
const side = props.sender === 'Vladimir' ? 'local' : 'remote'
```